src/ActionInputValidator/ActionValidators/PublishProfileContainerWebAppValidator.ts

```diff
@@ -0,0 +1,24 @@
+import { packageNotAllowed, multiContainerNotAllowed, startupCommandNotAllowed, validateSingleContainerInputs } from "../Validations";
+
+import { ActionParameters } from "../../actionparameters";
+import { IValidator } from "./IValidator";
+
+export class PublishProfileContainerWebAppValidator implements IValidator {
+    
+    async validate(): Promise<void> {
```

Comment:
public async

Reply:
public by default - https://www.typescriptlang.org/docs/handbook/classes.html#public-by-default

Reply:
It is just that we have been using public async in all actions. Wanted to keep it in sync. :)